docs/src/formula.md

```diff
@@ -40,8 +40,7 @@ Because it's often convenient to include main effects and interactions for a
 number of variables, the `*` operator will expand in this way
 
 ```jldoctest
-# Using the internal StatsModels.Terms to trigger expansion of the Formula...
-julia> Formula(StatsModels.Terms(y ~ 1 + x*y))
+julia> Formula(StatsModels.Terms(y ~ 1 + x*y)) # Parse by converting to Terms
```

Comment:
The old comment was more explicit. Maybe keep it, but moving it outside of the code block?